.pre-commit-config.yaml

```diff
@@ -108,6 +108,36 @@ repos:
     hooks:
       - id: actionlint-docker
 
+  - repo: https://github.com/codespell-project/codespell
+    rev: v2.2.5
+    hooks:
+      - id: codespell
+        args: [
+            # Pass skip configuration as command line arguments rather than in the
+            # config file because neither cfg nor TOML support splitting this argument
+            # across multiple lines.
+            "-S",
+            "sample_data/*",
+            "-S",
+            "frontend/test/tapes/**/*",
+            "-S",
+            "frontend/test/playwright/e2e/resources/*",
+            "-S",
+            "frontend/test/unit/fixtures/*",
+            "-S",
+            "catalog/tests/dags/providers/provider_api_scripts/resources/**/*",
+            "-S",
+            "catalog/tests/dags/common/test_resources/**/*",
```

Comment:
There is a spelling correction in a test resource file in this PR, that might not be necessary with this skip.

Side-note/tangent: I'm not able to mentally track which file/directory is skipped by which tool so it might be a good idea to either have some docs reference for skipping or have the skips be defined in a common place.

Reply:
I got confused about this today as well with the ESLint configuration. Agreed that additional documentation here would be useful.